packages/nimble-components/src/mapping/base/index.ts

```diff
@@ -0,0 +1,16 @@
+import { attr, ViewTemplate } from '@microsoft/fast-element';
+import { FoundationElement } from '@microsoft/fast-foundation';
+
+/**
+ * Maps values to a template
```

Comment:
Done

Reply:
> Related: Should we consider moving this somewhere with "table" in its name? The class includes table concepts like cells and group headers and it is only usable with specific table columns, so it feels a bit weird to be listed as a top-level component without that context. Maybe we should rename the `mapping` directory to `table-column-mapping`? Or move these under `table-column/mappings`?

I originally had `table-column\mapping\mappings`, but Milan (reasonably) suggested I move them directly under `src`. I don't want to rename the mapping directory to `table-column-mapping` because that sounds like it is a table column type (and it actually does collide with an existing table column type).

> Part of the challenge is that we are using the same "mapping" terminology to refer to both the element that maps a single value and some of the column types that contain those mappings.

Do you think we should rename `nimble-table-column-mapping` to `nimble-table-column-mapped-text`? Even if we do, that doesn't solve the issue of where to put the mapping elements. It still feels wrong to put them in a `table-column\mapping` directory, because it would look like another column type. I feel like putting them directly under `src` isn't that much different than putting `list-option` directly under `src`. That said, because they are not shared between the two mapping column types (i.e. mapping and icon), we could put `mapping-text` under `table-column\mapping` and `mapping-icon`/`mapping-spinner` under `table-column\icon`. Maybe in a file right next to `index.ts`? Thoughts?

Reply:
We talked and settled on:
- `src/mapping/text`, etc. with elements named `nimble-mapping-text/icon/spinner`
- `nimble-table-column-enum-text` instead of `nimble-table-column-mapping`

Reply:
Could you update the spec to reflect this change? I don't need them to stay up to date with every little API change, but keeping the name of the element in sync is important for understandability.